### PR TITLE
Cr changes1

### PR DESCRIPTION
**Pull request by Creatorbot01**
# Description

Added the glasses version for most common hud and few tweak

# Changelog

:cl:
- add: Added medical, medsec, and agent glasses
- add: Added medical and medsec glasses to medical and corpsman loadouts

### CODE REVIEW

File: Resources/Prototypes/_Arc/Entities/clothing/eyes/arcglasses.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    - Biological
+  - type: ShowJobIcons
+  - type: ShowMindShieldIcons
+  - type: ShowCriminalRecordIcons
+
+- type: entity
+  parent: ClothingEyesBase
+  id: ClothingEyesGlassesCR
+  name: agent glasses
+  description: Specialize glasses for agent in mission that provide flash and eyes protection and medical, security and diagnostic HUD.
```
Comment (TheDoctor1977) [inline Resources/Prototypes/_Arc/Entities/clothing/eyes/arcglasses.yml:65]:
```suggestion
  description: Highly advanced glasses for Central Command personnel that provide flash immunity and all three HUD types.
```

**Review by TheDoctor1977 — CHANGES_REQUESTED** [review]
minor issues

File: Resources/Prototypes/Loadouts/Jobs/Security/corpsman.yml [hunk 2]
```diff
@@ -58,7 +58,33 @@
...
+    - !type:CharacterJobRequirement
+      jobs:
+        - Brigmedic
+  items:
+    - ClothingEyesGlassesMedSec
 
+- type: loadout
+  id: LoadoutClothingEyesHUDMedSec
+  category: JobsSecurityCorpsman
+  cost: 0
```
[the remark this answers is not in the document]
Comment (Creatorbot01) [reply Resources/Prototypes/Loadouts/Jobs/Security/corpsman.yml:78]: tbh this is easily can be made by corpsman. this can make this round start easily. like (sechud, sec radio) from secvend. and medhud from nanovend. and wire, capacitor on that one machine i forgot.

beside they are suppose to be healer for sec so having medsec hud easily arent that bad.

[the remark this answers is not in the document]
Comment (Creatorbot01) [reply Resources/Prototypes/Loadouts/Jobs/Security/corpsman.yml:78]: but il up the price to one just to satified ya

File: Resources/Prototypes/_Arc/Entities/clothing/eyes/arcglasses.yml (hunk 1 above)
Comment (Creatorbot01) [reply Resources/Prototypes/_Arc/Entities/clothing/eyes/arcglasses.yml:65]: the reason why i dont mention central command so that the item wont be centcom only. it so that we can just reuse it for future content or something. 
tldr: i dont wanna add new glasses for each job that arent related to centcom that also need to have all hud thingi.

but il change it a bit cause agent in mission sound awful in my opinion

Comment (Creatorbot01) [reply Resources/Prototypes/_Arc/Entities/clothing/eyes/arcglasses.yml]: also this thing are able to be use for welding hence eye protection.

**Review by TheDoctor1977 — DISMISSED** [review]
couple of minor nitpicks remaining otherwise this is all good

File: Resources/Prototypes/_Arc/Entities/clothing/eyes/arcglasses.yml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    - Biological
+  - type: ShowHealthIcons
+    damageContainers:
+    - Biological
+
+- type: entity
+  parent: ClothingEyesBase
+  id: ClothingEyesGlassesMedSec
+  name: medsec glasses
+  description: Upgraded sunglasses that provide flash immunity and a medical HUD and security HUD.
```
[the remark this answers is not in the document]
Comment (Creatorbot01) [reply Resources/Prototypes/_Arc/Entities/clothing/eyes/arcglasses.yml:35]: i prefer to call it medical HUD and security HUD. yeah i dont like putting medsec hud because it sounded lazy as fuck. beside it was almost the same as medsec hud description so yeah im not changing this one.

**Review by TheDoctor1977 — APPROVED** [review]
i can’t spot anything too wrong here
@Erisfiregamer1 give this a look and then it should be good to merge

**Review by Erisfiregamer1 — DISMISSED** [review]
Why did you replace the sec glasses/ mesons in most of the loadouts with CR glasses???

**Comment by Creatorbot01** [comment]
> Why did you replace the sec glasses/ mesons in most of the loadouts with CR glasses???
ert dude. cr glasses are agent glasses. it basicly both secmed glass with weld protection.
a little ert buffed.

**Comment by Creatorbot01** [comment]
only ert and centcom get the glasses in their loadout rn.

**Comment by Creatorbot01** [comment]
since everything seem good with 3 weird fail imma merge it

**Comment by Erisfiregamer1** [comment]
I still see zero reason the loadouts should be updated, that just makes every merge more hellish if EE decides to modify loadouts.

If this starts causing issues I'm reverting it immediately.
